src/app/@core/utils/themes.service.ts

```diff
@@ -3,6 +3,7 @@ import { NbAuthService } from '@nebular/auth';
 import { NbLayoutDirection, NbLayoutDirectionService, NbThemeService } from '@nebular/theme';
 import { map, tap } from 'rxjs/operators';
 import { UserData } from '../interfaces/users';
+import { MediaMatcher } from '@angular/cdk/layout';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/HEPIA-Association/association-WebClient/pullRequest?prid=7914646)
```suggestion
import { MediaMatcher } from "@angular/cdk/layout";
```